crates/driver/src/domain/competition/auction.rs

```diff
@@ -112,6 +113,24 @@ impl Auction {
     pub fn score_cap(&self) -> Score {
         self.score_cap
     }
+
+    pub fn normalized_prices(&self) -> HashMap<eth::TokenAddress, auction::NormalizedPrice> {
```

Comment:
Should this return a new type?
Asking because we pass around a bunch of these and it's not obvious if those prices are reference prices coming from the auction, clearing prices or adjusted clearing prices.

Reply:
Added type alias.